### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Tracim/Frontend](https://hosted.weblate.org/projects/tracim/frontend/).

It also includes following components:

* [Tracim/Frontend App Html Document](https://hosted.weblate.org/projects/tracim/frontend-app-html-document/)

* [Tracim/Frontend App Folder Advanced](https://hosted.weblate.org/projects/tracim/frontend-app-folder-advanced/)

* [Tracim/Frontend App Kanban](https://hosted.weblate.org/projects/tracim/frontend-app-kanban/)

* [Tracim/Frontend App Collaborative Document Edition](https://hosted.weblate.org/projects/tracim/frontend-app-collaborative-document-edition/)

* [Tracim/Frontend App Gallery](https://hosted.weblate.org/projects/tracim/frontend-app-gallery/)

* [Tracim/Frontend App Share Folder Advanced](https://hosted.weblate.org/projects/tracim/frontend-app-share-folder-advanced/)

* [Tracim/Frontend App Thread](https://hosted.weblate.org/projects/tracim/frontend-app-thread/)

* [Tracim/Frontend App Workspace](https://hosted.weblate.org/projects/tracim/frontend-app-workspace/)

* [Tracim/Backend](https://hosted.weblate.org/projects/tracim/backend/)

* [Tracim/Backend - Default Custom Properties](https://hosted.weblate.org/projects/tracim/backend-default-custom-properties/)

* [Tracim/Frontend App File](https://hosted.weblate.org/projects/tracim/frontend-app-file/)

* [Tracim/Frontend Lib](https://hosted.weblate.org/projects/tracim/frontend-lib/)

* [Tracim/Frontend App Admin Workspace User](https://hosted.weblate.org/projects/tracim/frontend-app-admin-workspace-user/)

* [Tracim/Frontend App Agenda](https://hosted.weblate.org/projects/tracim/frontend-app-agenda/)

* [Tracim/Frontend App Workspace Advanced](https://hosted.weblate.org/projects/tracim/frontend-app-workspace-advanced/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: frontend/i18next.scanner/fr/translation.json [hunk 1]
```diff
@@ -518,32 +518,32 @@
...
+  "April": "Avril",
+  "May": "Mai",
+  "June": "Juin",
+  "July": "Juillet",
+  "August": "Août",
+  "September": "Septembre",
+  "October": "Octobre",
+  "November": "Novembre",
+  "December": "Decembre",
+  "MM/dd/yyyy": "jj/mm/aaaa",
```
Comment (raphj) [inline frontend/i18next.scanner/fr/translation.json:547]: Not sure about this one. Shouldn't "MM/dd/yyyy" be translated  "mm/jj/aaaa"? Or is the format adapted here?

File: frontend/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -518,32 +518,32 @@
...
+  "March": "Mars",
+  "April": "Avril",
+  "May": "Mai",
+  "June": "Juin",
+  "July": "Juillet",
+  "August": "Août",
+  "September": "Septembre",
+  "October": "Octobre",
+  "November": "Novembre",
+  "December": "Decembre",
```
Comment (raphj) [inline frontend/i18next.scanner/fr/translation.json:546]:
```suggestion
  "December": "Décembre",
```

File: frontend/i18next.scanner/fr/translation.json (hunk 1 above)
Comment (raphj) [reply frontend/i18next.scanner/fr/translation.json:547]: and if this format is used to display a date, the original letters should probably be kept like this:

```suggestion
  "MM/dd/yyyy": "dd/MM/yyyy",
```

**Comment by gbonaspetti** [comment]
@raphj fix on weblate

**Review by raphj — APPROVED** [review]

**Pull request merged**